src/shapes/Circle.ts

```diff
@@ -169,12 +181,13 @@ export class Circle<
         startY = sin(start) * radius,
         endX = cos(end) * radius,
         endY = sin(end) * radius,
-        largeFlag = angle > 180 ? '1' : '0';
+        largeFlag = angle > 180 ? '1' : '0',
+        sweepFlag = this.counterclockwise ? '0' : '1';
```

Comment:
I would turn these values to integers.
A relic of the past preserved them as strings
And of course we can do `Number(!this.counterclockwise)` but that is not crucial for me at all

Reply:
I would prefer not to do anything tricky like that.